src/simplewallet/simplewallet.cpp

```diff
@@ -104,10 +104,6 @@ typedef cryptonote::simple_wallet sw;
 
 #define EXTENDED_LOGS_FILE "wallet_details.log"
 
-#define DEFAULT_MIX DEFAULT_MIXIN
-
-#define MIN_RING_SIZE 12 // Used to inform user about min ring size -- does not track actual protocol
```

Comment:
`MIN_RING_SIZE` is here `12 == DEFAULT_MIXIN` but with this PR `get_min_ring_size()` is returning min_mixin + 1`. This means the new code is not equal to the old.